src/SFA.DAS.ApprenticeCommitments.Web/Controllers/ProfileController.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Web;
+using Microsoft.AspNetCore.Authorization;
+using SFA.DAS.ApprenticeCommitments.Web.Startup;
+
+namespace SFA.DAS.ApprenticeCommitments.Web.Controllers
+{
+    public class ProfileController : Controller
+    {
+        private readonly AuthenticationServiceConfiguration _authenticationConfig;
+
+        public ProfileController(AuthenticationServiceConfiguration authenticationConfig)
+        {
+            _authenticationConfig = authenticationConfig;
+        }
+
+        [Authorize]
+        [HttpGet("/profile/{clientId}/changeemail/confirm")]
```

Comment:
Should we remove the `{clientId}` from this URL in the ADC web?  It seems a little odd to include it, as ADC *is* the client - and already has a client ID embedded.  The `clientId` used in this method is actually the Login Service's internal database ID for the client. I think it would be preferrable to keep that within the login service, and not expose it.

It would improve the URL seen by the user in their email as well - as the ugly GUID would be removed.

It would also mean that the client is responsible for providing the same client ID in all interractions with the login service, rather than a public client ID for most, and a GUID for one.

Reply:
Discussed with team, will defer this to when we add the link to initiate Change of Password, as they will both require a Client ID of some kind.